web-ui/src/main/resources/catalog/components/viewer/owscontext/OwsContextService.js

```diff
@@ -324,6 +326,13 @@
             firstLoad = false;
           }
         }
+        
+        // broadcast context load
+        $rootScope.$broadcast('owsContextLoaded');
```

Comment:
Dummy promise, but makes easier to handle this function the same way as if it was a loadContextByUrl.

Reply:
Moved event to the lower end of the function, which is when the context has finished loading